docs/USING.rst

```diff
@@ -75,10 +75,11 @@ Included herewith are some examples about
 1) Register a switch with HIL:
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Eg> Switch name: mockswitch01
-     Host name:  switchhost01
-     User name:  switchuser01
-     Password:   password1234
+   - **Switch name:** mockswitch01
+   - **Switch type:** mock
+   - **Host name:**   switchhost01
+   - **User name:**   switchuser01
+   - **Password:**    password1234
```

Comment:
This is currently being rendered as a block comment; you should un-indent it. There are a couple of other instances of this problem elsewhere.

Reply:
Was doing that on purpose since that's what one of the examples did previously. Hmm, I think you ar right and it doesn't make sense to block comment them.